lib/models/application.coffee

```diff
@@ -103,7 +103,11 @@ getApplicationModel = (deps, opts) ->
 					mergePineOptions
 						$orderby: 'app_name asc'
 						$filter:
-							user: userId
+							$or:
```

Comment:
I thought `$or` always took an array? FBM if you've tested it, this is neat.

Reply:
Also tested and works fine.